src/util.js

```diff
@@ -14,12 +14,17 @@ const focusableElementsList = [
     '*[contenteditable]'
 ];
 
-function createTrapBoundary() {
-    const div = document.createElement('div');
-    div.setAttribute('tabindex', '0');
-    div.className = 'keyboard-trap-boundary';
+let trapBoundary;
```

Comment:
I like the idea of cloning this node, but perhaps we should instantiate it lazily? Not a huge deal I know, but it might be better than running noop.